static/app/views/issueDetails/actions/index.spec.tsx

```diff
@@ -177,6 +177,34 @@ describe('GroupActions', function () {
     });
   });
 
+  it('delete for issue platform is enabled', async () => {
+    const org = OrganizationFixture({
+      ...organization,
+      access: [...organization.access, 'event:admin'],
+      features: [...organization.features, 'issue-platform-deletions-ui'],
+    });
+    render(
+      <Fragment>
+        <GlobalModal />
+        <GroupActions
+          group={group}
+          project={project}
+          organization={org}
+          disabled={false}
+        />
+      </Fragment>,
+      {organization: org}
+    );
+
+    await userEvent.click(screen.getByLabelText('More Actions'));
+    expect(
+      await screen.findByRole('menuitemradio', {name: 'Delete'})
+    ).not.toHaveAttribute('aria-disabled');
+    expect(
+      await screen.findByRole('menuitemradio', {name: 'Delete and discard future events'})
+    ).toHaveAttribute('aria-disabled');
```

Comment:
I'm confused as to why this does not pass. The test runner does not give me info as to what attributes it has.
<img width="809" alt="image" src="https://github.com/user-attachments/assets/3f79d7d1-7a81-4f62-b8ff-69ce804ce788">

![image](https://github.com/user-attachments/assets/45e4d317-4fe9-4628-8f54-51abfcdd415d)